hello-world-master/source/components/scene/animationhelper.brs

```diff
@@ -0,0 +1,35 @@
+function translationAnimation(target as Object, originX as Integer, originY as Integer, newX as Integer, newY as Integer) as Object
```

Comment:
We can name this "createTranslationAnimation" because the current name may be nissleading

Reply:
We can name `origin` and `destination` for parameters